Gemfile

```diff
@@ -9,12 +9,14 @@ gem 'haml'
 gem 'sass'
 
 gem 'sqlite3'
+gem 'rqrcode'
 
 group :development do
   gem 'byebug'
 end
 
 group :test do
+  gem 'rspec'
```

Comment:
What's your reason for adding rspec to the project? Minitest is already available.